docs/core/PORTING.md

```diff
@@ -380,16 +420,27 @@ SECTIONS
         __uvisor_bss_boxes_end = .;
 
         /************************** ARMv7-M MPU only **************************/
-        __uvisor_bss_end_padding_max = (2 << (LOG2CEIL(__uvisor_bss_end - ORIGIN(m_data)) - 1)) / 8;
-        . = MIN(
-            __uvisor_bss_end_padding_max * (((__uvisor_bss_end - ORIGIN(m_data)) / __uvisor_bss_end_padding_max) +
-            MIN((__uvisor_bss_end - ORIGIN(m_data)) % __uvisor_bss_end_padding_max, 1)) - __UVISOR_SRAM_OFFSET,
-            ORIGIN(m_data) + LENGTH(m_data)
-        );
+        . = ALIGN((1 << LOG2CEIL(LENGTH(m_data))) / 8);
```

Comment:
This alignment isn't needed, the page allocator can make use of this memory and fit a few more pages in.